app/controllers/reducers_controller.rb

```diff
@@ -131,4 +131,17 @@ def reducer_params(klass)
   def record_not_valid(exception)
     render json: { error: exception.message }, status: 422
   end
+
+  def convert_to_array(input)
+    if input.strip.start_with?("[") && input.strip.end_with?("]")
+      json_string = input.gsub("'", '"')
```

Comment:
I think this could line lead to weird behavior. As an eg. someone enters a single quote within a double quote within an array eg inputting ["'hello'"] . I think as you edit the reducer, this logic starts getting wonky. 

Granted a person shouldnt be doing something like that anyway cuz I'm assuming the reducer won't run well  😆 , but technically via caesar ui is still a "valid" input.